homeassistant/components/launch_library/sensor.py

```diff
@@ -163,10 +164,13 @@ def __init__(
     ) -> None:
         """Initialize a Launch Library sensor."""
         super().__init__(coordinator)
-        if name:
-            self._attr_name = name
         self._attr_unique_id = f"{entry_id}_{description.key}"
         self.entity_description = description
+        self._attr_device_info = DeviceInfo(
+            identifiers={(DOMAIN, entry_id)},
+            entry_type=DeviceEntryType.SERVICE,
+            name=name,
```

Comment:
```suggestion
            name=name or description.name,
```
This would be better here?

Reply:
The idea behind this change is to add a device to the integration. With this change, the resulted entity_id's would be `{name} Next Launch`. In a follow up PR I will add the translations.

And since there can only be one instance of this integration on a system, why should we rename one entity for the old YAML people, whilst now everyone can rename the device to whatever they like and all entities will change?

Reply:
To not add unessesarry breaking changes for legacy users.
Those are hard and confusing to document/describe as it would only affect a small subset of users.

Reply:
But the entites have an unique_id, how will this be a breaking change?